src/app/individual/individual.ts

```diff
@@ -29,3 +30,31 @@ export class Individual {
   type: IndividualType;
   image_urls: string[];
 }
+
+export class MapIndividual implements IdLike {
+  id: string;
+  geopos: google.maps.LatLngLiteral;
+  source: SourceType;
+  species?: string;
+  station_species?: string[];
+  last_phenophase?: string;
+  type: IndividualType;
+
+  constructor(
```

Comment:
check to use interface instead of class. We do not use class constructors anywhere in the project.